lib/text-formatters.js

```diff
@@ -53,4 +49,22 @@ function omitv(version) {
   }
   return version;
 }
-exports.omitv = omitv;
+
+function maybePluralize(singular, countable, plural) {
+  plural = plural || `${singular}s`;
```

Comment:
I'm not usually a fan of logic like this as it doesn't handle languages other than English, but I guess our labels are all English so it's probably OK. If you wanted to be very fancy, you could use an inflection library for this (eg. [node.inflection](https://github.com/dreamerslab/node.inflection)) which would avoid needing to have the `plural` argument, as it properly handles all pluralization.

Reply:
I do feel like this codebase tends to reimplement this kind of stuff when it should just use a library. I'm fine either way, TBH. If you can vouch for node.inflection I'd support dropping it in as a replacement.

Reply:
It's just a library I found via Google, I've never actually used it. Looked through the code and it seems legit though.